marketing team additions (#4875)
kubernetes/community: communication/marketing-team/README.md

| Area & Handbook Link | Lead | Team | Notes |
| --- | --- | --- | --- |
| [Editorial] | editor: @mbbroberg | assoc editor: [Could be you!] | advisor: @parispittman |
| [Internal Communications] | @jonasrosland | @rajula96reddy |  |

> | [Social Media] | @chris-short @kaslin | n/a | |

Reviewer: ```suggestion
| [Social Media] | @chris-short @kaslin | | |
```

I'm pro leaving this blank as opposed to introducing `n/a`

- Reach out to an individual above via slack
- Whole group - @contributor-comms in #sig-contribex on slack
- Tag us in an issue in [kubernetes/community] with the label [area/contributor-comms]

> - Set a marketing issue template in [kubernets/community]

Reviewer: ```suggestion
- Use the [Contributor Comms Request](https://github.com/kubernetes/community/issues/new/choose) issue template in [kubernets/community]
```

> - Follow us on @K8sContributors on Twitter

Reviewer: ```suggestion
- Follow us on [@K8sContributors](https://twitter.com/K8sContributors) on Twitter
```
